src/main/main.cpp

```diff
@@ -0,0 +1,9 @@
+#include <iostream>
+#include "config.hpp"
+
+int main(int argc, char *argv[])
+{
+	Config config;
+	config.detect_usecase(argc, argv);
+	return 0;
```

Comment:
An exit value of 0 in the unix world means "everything worked well" but there are cases when hawk can fail (for example if the provided config is incorrect) it is useful to return a negative number in those cases to let the user know that hawk failed (you might not see the immediate value of this when running hawk manually but this will be really helpful when users will put hawk in bash scripts)

Reply:
Other cases would be when loading the eBPF program failed, this error deserves to be propagated back